scripts/eateries/Eatery.js

```diff
@@ -0,0 +1,7 @@
+export const Eatery = (eateryObj) => {
+    return `<section class="preview__card eatery__card">
+    <h3 class="eatery__name">${eateryObj.fullName}</h3>
```

Comment:
Line 3: ```${eateryObj.fullName}``` does not match the key:value name in the Eateries object. Needs to be renamed to ```${eateryObj.businessName}```.